sycl/source/CMakeLists.txt

```diff
@@ -143,8 +165,13 @@ if (MSVC)
 
   target_compile_options(sycl PRIVATE ${SYCL_CXX_FLAGS_RELEASE})
 
-  add_sycl_rt_library(sycld ${SYCL_SOURCES})
+  if (SYCL_ENABLE_XPTI_TRACING AND CMAKE_BUILD_TYPE MATCHES "Debug")
```

Comment:
Why do you check for CMAKE_BUILD_TYPE here?

Reply:
I was assuming that each build type had a separate pass through make - and it looks like both Debug and Release are built with the same configuration. Made changes that builds without failure now.